src/main/java/nextstep/ladder/domain/Line.java

```diff
@@ -33,4 +36,32 @@ public static Line of(int countOfPerson) {
     public List<Point> points() {
         return new ArrayList<>(points);
     }
+
+    private boolean moveLeft(int curPosition) {
```

Comment:
> 규칙 5: 줄여쓰지 않는다(축약 금지)
```suggestion
    private boolean moveLeft(int currentPosition) {
```

논리 타입을 리턴하니 메서드명을 형용사로 변경하면 어떨까요?
메서드를 호출하는 코드에서도 자연스러울 것 같아요